src/ontodia/diagram/halo/haloMarkup.tsx

```diff
@@ -0,0 +1,49 @@
+import * as joint from 'jointjs';
```

Comment:
I don't think that it makes sense to split halo implementation across two files (`halo.ts` and `haloMarkup.tsx`).
Also, `src/ontodia/viewUtils` will be a better place for Halo.